packages/bierzo-wallet/src/utils/tokens.ts

```diff
@@ -60,3 +62,21 @@ export async function chainAddressPairSortedMapping(
 
   return chainAddresses;
 }
+
+export function getErc20TokensConfig(options?: ConfigEthereumOptions): Erc20TokensMap | undefined {
+  if (!options) return undefined;
```

Comment:
To simplify the signature of this function, I'd make `options` non-optional. Then just call it conditionally:

```ts
erc20Tokens: spec.ethereumOptions ? getErc20TokensConfig(spec.ethereumOptions) : undefined,
```